### PR TITLE
py: Fix TurboSHAKE tests

### PR DESCRIPTION
**Pull request by cjpatton**
The customization string is now a customization byte.

### CODE REVIEW

File: py/TurboSHAKE.py [hunk 1]
```diff
@@ -86,4 +86,4 @@ def TurboSHAKE128(M, D, outputByteLen):
     return TurboSHAKE(256, M, D, outputByteLen)
 
 def TurboSHAKE256(M, D, outputByteLen):
-    return TurboSHAKE(512, M, D, outputByteLen)
\ No newline at end of file
+    return TurboSHAKE(512, M, D, outputByteLen)
```
Comment (cjpatton) [inline py/TurboSHAKE.py]: My editor prefers a newline. Let me know if you'd like me to revert.

**Comment by jedisct1** [comment]
Looks good, and consistent with the `TurboSHAKE` function definitions.

File: py/TurboSHAKE.py (hunk 1 above)
Comment (jedisct1) [reply py/TurboSHAKE.py]: Rather than editor-specific settings or personal preferences, may I suggest using a formatter such as Black?

Comment (cjpatton) [reply py/TurboSHAKE.py]: Done!

**Review by jedisct1 — APPROVED** [review]

**Review by ildyria — APPROVED** [review]

**Pull request merged**